package.json

```diff
@@ -59,7 +59,8 @@
     "TODO: remove validated-changeset resolution when https://github.com/poteto/ember-changeset/issues/650 is addressed"
   ],
   "resolutions": {
-    "**/validated-changeset": "~1.3.2"
+    "**/validated-changeset": "~1.3.2",
+    "**/highlight.js": "10.7.2"
```

Comment:
Can you add a comment explaining the downgrade similar to how it’s done for validated-changeset? A link to a GitHub issue would be ideal. If no issue exists until now, maybe open one?

Reply:
Done